dev/nuxt/pages/index.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <h2>vue-tinybox demo: Nuxt.js</h2>
+    <p>Click on any image below to open <code>Tinybox</code></p>
+
+    <client-only>
+      <Tinybox
+        v-model="index"
+        :images="images"
+        no-thumbs
+      />
+    </client-only>
+    <img
+      v-for="(img, idx) in thumbs"
+      class="open-tinybox"
+      width="100"
+      :src="img"
+      @click="index = idx"
+    >
+  </div>
+</template>
+
+<script>
+
+import Tinybox from '../components/tinybox';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../components/tinybox"](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=4749911)